modules/ghlint-reporting-sarif/src/main/kotlin/net/twisterrob/ghlint/reporting/sarif/SarifReporter.kt

```diff
@@ -100,7 +100,7 @@ private fun reportingDescriptor(issue: Issue): ReportingDescriptor =
 	)
 
 private fun result(finding: Finding, base: Path): Result {
-	val file = Path.of(finding.location.file.path).absolute()
+	val file = Path.of(finding.location.file.path).toRealPath(LinkOption.NOFOLLOW_LINKS)
```

Comment:
@timsearle this is the fix.

Can you please run `gradlew :ghlint-reporting-sarif:build` to verify locally?

Reply:
![Screenshot 2024-06-02 at 17 35 04](https://github.com/TWiStErRob/net.twisterrob.ghlint/assets/5477014/866ebef9-8ed5-445f-a4b0-efe275f11378)
![Screenshot 2024-06-02 at 17 34 34](https://github.com/TWiStErRob/net.twisterrob.ghlint/assets/5477014/790b5419-0fbf-45d1-9527-b311f84660b3)